### ISSUE TITLE
Force Update Language mechanism

### ISSUE THREAD

**Issue by rg9400** [Radarr/Radarr#7787]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

Right now, there is no way to rerun the language selection in the Arrs for existing files. This means that as new languages are added to each one, existing files will still be mapped as English. Consider the problem case where you have a language profile set to Original or a "Wrong Language" CF tracking if the file language matches the original language of the movie. Maybe the original language of the movie is Tamil, which is not currently supported by Radarr, so both file language and original language will be English right now and matching. If Radarr adds support for Tamil in the future, the original language will update on refresh to Tamil, but the existing file language will remain as English. This suddenly is not matching, and thus will trigger the CF or potential false upgrades due to the language profile.

This is even more problematic in Sonarr v4 where the language parsing was not using AudioLanguages before, so almost all existing historical files will be currently mapped as English, regardless of their actual language.

### Describe the solution you'd like

A button in advanced settings that forces Radarr/Sonarr to rerun through the Language Selection logic for either a specific series/movie or the entire library. This would use existing MediaInfo rather than rescanning the file and just rerun the logic as if the file was being imported again. You would have to override the existing languages even if the user had manually edited them at some point as the Arrs will not be able to recognize it, but this would remain as an Advanced Setting for this reason.

### Describe alternatives you've considered

There is no way to force a refresh like you can do for MediaInfo by manually editing the MI Schema Version down in the DB (though I think having a button to force rescan MI would be useful still as DB editing is dangerous). The only way to do this right now is to write a script to try and do it via API. I tried to do so, but it is incredibly complex and inefficient. You need to list out every file, pull the MediaInfo for each file, parse out AudioLanguages, separate out each track, collate unique languages, map the MI language code to a language name, map that language name to supported Radarr languages (and dump unsupported languages into English), and then create a PUT request that just keeps everything else about the file same except the updated language.

In other words, this is recreating the existing Radarr logic for language selection in script format. And this requires constant maintenance as the mapping of languages has to be updated as Radarr updates their languages. Doing this per file for Sonarr would require a huge amount of time as it has to run per EpisodeFile. Having a native way to do it in Radarr is a much cleaner, safer, and efficient solution.

### Anything else?

No

**Comment by bakerboy448** [comment]
Manual Import from the movie page should force the rescan, no?

**Comment by rg9400** [comment]
> Manual Import from the movie page should force the rescan, no?

This works in a case by case basis, but to do this for every single movie/series is not feasible.
